c2cgeoportal/scaffolds/create/deploy/hooks/pre-restore-database.mako

```diff
@@ -11,23 +11,26 @@ DATABASES=$@
 # databases / tables restore.
 sudo apache2ctl stop
 
+if psql -lqt | cut -d \| -f 1 | grep -qw $DATABASES; then
```

Comment:
for  ´sudo -u postgresql´ your right :-)

> I can remove the q option if you don't like it

It's not that I don't like it but I prefer to don't have unneeded argument :-)

+1